producer-backend/producer.js

```diff
@@ -8,6 +9,23 @@ const kafka = new Kafka({
   brokers: [process.env.KAFKA_BOOTSTRAP_SERVER || 'my-cluster-kafka-bootstrap:9092']
 });
 
+let kafkaConnectionBindings;
+try {
+  // check if the deployment has been bound to a kafka instance through
+  // service bindings. If so use that connect info
+  kafkaConnectionBindings = serviceBindings.getBinding('KAFKA', 'kafkajs');
```

Comment:
Ideally this might return some of the well typed (typescript) values that users uses in their config. SBO will not cover all config options etc.